developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -132,6 +132,31 @@ All rake tasks are available with the following command:
 bundle exec rake -T
 ....
 
+[[plugins]]
+== Installing plugins
+In order to use a plugin, you'll need to install its gem.
+
+From source code:
+[source, ruby]
+....
+echo "gem '<PLUGIN_NAME>', path: '../PLUGIN_PATH'" >> bundler.d/<PLUGIN_NAME>.local.rb
+....
+
+From github:
+[source, ruby]
+....
+echo "gem '<PLUGIN_NAME>', git: 'git://github.com/theforeman/<PLUGIN_NAME>'" >> bundler.d/<PLUGIN_NAME>.local.rb
```

Comment:
I think it's supposed to be:
`echo "gem '<PLUGIN_NAME>', git: 'https://github.com/theforeman/<PLUGIN_NAME>.git'" >> bundler.d/<PLUGIN_NAME>.local.rb`